docs/api-reference.md

```diff
@@ -0,0 +1,439 @@
+Packages:
+
+-   [maintenance.nvidia.com/v1alpha1](#maintenance.nvidia.com%2fv1alpha1)
+
+## maintenance.nvidia.com/v1alpha1
+
+Package v1alpha1 contains API Schema definitions for the maintenance.nvidia.com v1alpha1 API group
+
+Resource Types:
+
+### DrainSpec
+
+(*Appears on:*[NodeMaintenanceSpec](#maintenance.nvidia.com/v1alpha1.NodeMaintenanceSpec))
```

Comment:
It looks like these links are broken? Is it possible to fix them somehow?

Reply:
yea, its not that smooth, lemme try to fix that.

you can specify lua filters to pandoc but im not an experet in neither :\.